wagtail/core/models.py

```diff
@@ -152,15 +152,16 @@ def clean_fields(self, exclude=None):
     @staticmethod
     def get_site_root_paths():
         """
-        Return a list of (root_path, root_url) tuples, most specific path first -
-        used to translate url_paths into actual URLs with hostnames
+        Return a list of (id, root_path, root_url) tuples, most specific path
+        first - used to translate url_paths into actual URLs with hostnames
         """
         result = cache.get('wagtail_site_root_paths')
 
         if result is None:
             result = [
                 (site.id, site.root_page.url_path, site.root_url)
-                for site in Site.objects.select_related('root_page').order_by('-root_page__url_path')
+                for site in Site.objects.select_related('root_page').order_by(
+                    '-root_page__url_path')
```

Comment:
Please revert this change. I know the line was too long, but since this change has no code consequence, please keep it the way it was.